src/common/models/Projection.ts

```diff
@@ -542,10 +517,10 @@ export class Projection {
   }
 
   private calcRtRange(
-    timeseries: Array<PredictionTimeseriesRow | null>,
+    timeseries: Array<MetricsTimeseriesRow | null>,
   ): Array<RtRange | null> {
-    const rtSeriesRaw = timeseries.map(row => row && row.RtIndicator);
-    const rtCiSeriesRaw = timeseries.map(row => row && row.RtIndicatorCI90);
+    const rtSeriesRaw = timeseries.map(row => row && row.infectionRate);
+    const rtCiSeriesRaw = timeseries.map(row => row && row.infectionRateCI90);
 
     // This hides small gaps (less than 2 data points) in the rt series to make
```

Comment:
FWIW- As part of this API v2 work, it would be nice to verify that none of this post-processing is needed anymore (we should fix it upstream if it is).  Though that doesn't need to be in this PR.

Reply:
done!